### ISSUE TITLE
Load external textures from the same path as the 3d model

### ISSUE THREAD

**Issue by impromedia** [kovacsv/Online3DViewer#388]
**Load external textures**

**Comment by kovacsv** [comment]
You can specify textures one by one in the url regardless of their location. Textures are identified by name, so they can be at any folder at any server.

For example:
https://3dviewer.net/#model=https://example.com/models/model.fbx,https://example.com/externaltexture.png

**Comment by impromedia** [comment]
Yes, but it is quite difficult to specify all the textures (and you must know them) for a complex model. If the engine will look in the same folder where is the mesh or just specify a folder for textures will be much effective, because it knows already the names of the textures, just location it is wrong. Thank you.

**Comment by FernandoJBCosta** [comment]
Hello, first of all good work, I have the same issue. I have a folder structure that has many projects and within each project there is a folder called textures, is it possible every time we open a file to look for the textures inside the /textures folder in each project?
Best regards

**Comment by FernandoJBCosta** [comment]
> You can specify textures one by one in the url regardless of their location. Textures are identified by name, so they can be at any folder at any server.
> 
> For example: https://3dviewer.net/#model=https://example.com/models/model.fbx,https://example.com/externaltexture.png

Hello. it's possible to force in the system a folder to search for the textures?
